src/test/java/trello/steps/CardSteps.java

```diff
@@ -71,4 +75,41 @@ public void verifyTheNameCard() {
         String actualTitle = boardPage.getCardTitle(context.getList().getTitle(), context.getCard().getTitle());
         Assert.assertEquals(context.getCard().getTitle(), actualTitle);
     }
+
+    /**
+     * Select a card of board.
+     *
+     * @param cardTitle defines the card title that want selected.
+     */
+    @And("I select (.*) card")
+    public void selectCard(final String cardTitle) {
+        boardPage = new BoardPage();
+        context.getCard().setTitle(cardTitle);
+        boardPage.selectedCard(cardTitle);
+    }
+
+    /**
+     * Add a checklist to card.
```

Comment:
Please, change the verb in third person.

Reply:
Solved